src/ExtendedTableWidget.cpp

```diff
@@ -270,7 +270,8 @@ QSet<int> ExtendedTableWidget::selectedCols()
 void ExtendedTableWidget::cellClicked(const QModelIndex& index)
 {
     // If Alt key is pressed try to jump to the row referenced by the foreign key of the clicked cell
-    if(qApp->keyboardModifiers().testFlag(Qt::ControlModifier) && qApp->keyboardModifiers().testFlag(Qt::ShiftModifier) && model())
+    if(qApp->keyboardModifiers().testFlag(Qt::ControlModifier) && qApp->keyboardModifiers().testFlag(Qt::ShiftModifier)
+       && model())
```

Comment:
I'd prefer to keep this line as it was. Since there is no code changed in this file, changing the formatting of this line could be confusing later: it looks like the code in this file was changed by this commit when in reality it wasn't. Hope this makes sense :grinning: